optaweb-employee-rostering-backend/src/main/java/org/optaweb/employeerostering/service/contract/ContractService.java

```diff
@@ -103,4 +103,19 @@ public Contract updateContract(Integer tenantId, ContractView contractView) {
         oldContract.setMaximumMinutesPerYear(newContract.getMaximumMinutesPerYear());
         return contractRepository.save(oldContract);
     }
+
+    @Transactional
+    public Contract getOrCreateDefaultContract(Integer tenantId) {
+        Optional<Contract> defaultContract = contractRepository.findAllByTenantId(tenantId)
+                .stream().filter(contract -> contract.getName().equals("Default Contract"))
+                .findAny();
+        if (defaultContract.isPresent()) {
+            return defaultContract.get();
+        } else {
+            Contract contract = new Contract();
+            contract.setName("Default Contract");
+            contract.setTenantId(tenantId);
+            return contractRepository.save(contract);
+        }
```

Comment:
The PR was created when we were merging features into the `covid-19` branch without testing. Since I have a separate PR that now add this testing to the `covid-19` branch, I will add tests to this PR.